test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -1254,6 +1254,58 @@ public void CopyColumnsOnnxTest()
             Done();
         }
 
+        [Fact]
+        public void FeatureSelectionOnnxTest()
+        {
+            var mlContext = new MLContext(seed: 1);
+
+            string dataPath = GetDataPath("breast-cancer.txt");
+
+            var dataView = ML.Data.LoadFromTextFile(dataPath, new[] {
+                new TextLoader.Column("ScalarFloat", DataKind.Single, 6),
+                new TextLoader.Column("VectorFloat", DataKind.Single, 1, 4),
+                new TextLoader.Column("VectorDouble", DataKind.Double, 4, 8),
+                new TextLoader.Column("Label", DataKind.Boolean, 0)
+            });
+
+            var columns = new[] {
+                new CountFeatureSelectingEstimator.ColumnOptions("FeatureSelectDouble", "VectorDouble", count: 1),
+                new CountFeatureSelectingEstimator.ColumnOptions("ScalFeatureSelectMissing690", "ScalarFloat", count: 690),
+                new CountFeatureSelectingEstimator.ColumnOptions("ScalFeatureSelectMissing100", "ScalarFloat", count: 100),
+                new CountFeatureSelectingEstimator.ColumnOptions("VecFeatureSelectMissing690", "VectorDouble", count: 690),
```

Comment:
>VecFeatureSelectMissing690 [](start = 66, length = 26)

Why such strange names?

Reply:
ah sorry you created these test columns


---
In reply to: [363882851](https://github.com/dotnet/machinelearning/pull/4562#discussion_r363882851) [](ancestors = 363882851)